web/packages/hovercards/src/style.scss

```diff
@@ -97,49 +102,81 @@ $color-blue: #1d4fc4;
 
 	.gravatar-hovercard__body {
 		min-height: 42px;
+		margin-top: 8px;
 	}
 
 	.gravatar-hovercard__description {
 
 		@include ellipsis(2);
 	}
 
-	.gravatar-hovercard__footer {
-		display: flex;
-		justify-content: space-between;
-		align-items: center;
+	.gravatar-hovercard__verified-icon {
+		padding: 4px;
+		box-sizing: border-box;
 	}
 
 	.gravatar-hovercard__social-links {
 		display: flex;
 		align-items: center;
-		gap: 8px;
+		gap: 4px;
+		margin-top: 16px;
 	}
 
-	.gravatar-hovercard__profile-link {
+	.gravatar-hovercard__buttons {
+		display: flex;
+		gap: 16px;
+		margin-top: 16px;
+	}
+
+	.gravatar-hovercard__button {
+		width: 100%;
+		min-height: 42px;
 		display: flex;
 		align-items: center;
-		text-decoration: none;
-		gap: 4px;
+		justify-content: center;
```

Comment:
Q: Do we need the `display`, `align-items`, and `justify-content` for `<button>` elements? Seems fine without these props.